src/classes/Entity/deep-copy.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'deep-copy' {
+    declare function dcopy<T>(arg: T): T
+    export default dcopy
+}
```

Comment:
just wondering,
why do we need this? deep-copy comes out of the box with type defs.
Not sure what I'm missing 🙏

Reply:
The type defs don't support ES modules. The export a function, not a module. ES modules require a module declaration for modules to be reference correctly.

The type def shipped looks like this

```js
declare function dcopy<T>(arg: T): T

export default dcopy
```

with no module declaration

Reply:
Oh, gotcha!